dockerfiles/verse_4.2.0.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL org.opencontainers.image.licenses="GPL-2.0-or-later" \
       org.opencontainers.image.authors="Carl Boettiger <cboettig@ropensci.org>"
 
 ENV CTAN_REPO=https://mirror.ctan.org/systems/texlive/tlnet
-ENV PATH=$PATH:/usr/local/texlive/bin/x86_64-linux
+ENV PATH=$PATH:/usr/local/texlive/bin/linux
```

Comment:
Do not edit this Dockerfile by hand.
Please check the document. https://github.com/rocker-org/rocker-versioned2/blob/ae741564c8c965672d329a94e7930c4e4ba98809/build/README.md

Reply:
> However, there may be no need to update these Dockerfiles at this stage as these can only be built on linux/arm64.

You meant linux/amd64, correct? Just looking ahead for future linux/arm64/v8 builds

> I also wonder if adding the symbolic link to a location already added to the PATH would eliminate the need to update the PATH.

No. The link – not the target – needs to be in PATH.

> Please check the document. https://github.com/rocker-org/rocker-versioned2/blob/ae741564c8c965672d329a94e7930c4e4ba98809/build/README.md

I will check and update the pull request accordingly.